exercises/perfect-numbers/perfect_numbers_test.rb

```diff
@@ -3,23 +3,10 @@
 require_relative 'example'
 
 class PerfectNumberTest < Minitest::Test
-  def test_aliquot_sum_zero
-    assert_equal 0, PerfectNumber.new(1).aliquot_sum
-  end
-
-  def test_aliquot_sum_prime
-    skip
-    assert_equal 1, PerfectNumber.new(2).aliquot_sum
-  end
-
-  def test_aliquot_sum_multiple_divisors
-    skip
-    assert_equal 15, PerfectNumber.new(16).aliquot_sum
-  end
-
-  def test_aliquot_sum_large_num
-    skip
-    assert_equal 229, PerfectNumber.new(1115).aliquot_sum
+  def test_initialize_perfect_number
+    assert_raises RuntimeError do
+      PerfectNumber.new(-1)
+    end
```

Comment:
I'm having trouble getting the assert_raises to work when I try to implement it in this format. NoMethodError: undefined method `RuntimeError' is the error I get.